osu.Game.Tests/Resources/TestResources.cs

```diff
@@ -13,8 +13,31 @@ public static class TestResources
 
         public static Stream OpenResource(string name) => GetStore().GetStream($"Resources/{name}");
 
-        public static Stream GetTestBeatmapStream(bool virtualTrack = false) => OpenResource($"Archives/241526 Soleily - Renatus{(virtualTrack ? "_virtual" : "")}.osz");
+        public static Stream GetTestBeatmapStream(bool virtualTrack = false, bool quick = false) => OpenResource($"Archives/241526 Soleily - Renatus{(virtualTrack ? "_virtual" : "")}{(quick ? "_quick" : "")}.osz");
```

Comment:
Bit of a nitpick, but if someone ever outwardly calls this as

```csharp
TestResources.GetTestBeatmapStream(virtualTrack: false, quick: true);
```

for some reason, it'll fail. Not sure it's super important.

Reply:
This is because that combination doesn't actually exist. I guess I'll just revert this and specify the quick one inline to avoid any potential misuse.